### DISCUSSION TITLE
Got too low payment from pool.

### DISCUSSION THREAD

**Discussion by vese** [Chia-Network/chia-blockchain#7411] [Support]
I joined cis.ecochia.io pool and got more than 10 times less payment than previous payments for the last found block in pool.
At that time I recieved points as usual (second image). Pool netspace did not increased by 10 times in 6 hours.
And using calculator can get number 8 times greater than payment (third image) for 7tb (pool netspace was 29.52 PiB).
Are this payments processed automatically by chia pool api or manually by pool holders?
![image](https://user-images.githubusercontent.com/17708516/125300001-4f1b6f00-e332-11eb-80d0-99b6498a9a3a.png)
![image](https://user-images.githubusercontent.com/17708516/125300108-64909900-e332-11eb-9d3c-9db6106514c2.png)
![image](https://user-images.githubusercontent.com/17708516/125301188-63ac3700-e333-11eb-88c4-6698fba38aa8.png)

**Comment by MaximusChia** [comment]
Your share must have been decreased due to other farmers surpassed you in points, if a whale joined pool you might expext less rewards but more luck to fin a block faster. Check spacefarmers.io pool dashboards it has well detailed dashboard so you can have a better idea of how shares work. ALso based on your graph you got considerably less points on the last 2 hours so theres other reason why your share is less...

**Reply by vese** [reply]
No one with extra 200PiB+ joined the pool with 30PiB. Left side of graph is related to the moment when pool found the block. I assume it was 150+ poinst for 8 hours after previous block. It might be less than 700/(24/3)≈230 points but not less in 8 times.

**Reply by vese** [reply]
Also other people with same number of plots in same pool got normal payments
